### ISSUE TITLE
'wandb.tensorboard.unpatch()' missing in documentation

### ISSUE THREAD

**Issue by daniel-bogdoll** [wandb/wandb#8937]
Hey everyone,

I log my experiments with Tensorboard and have multiple experiments per run. Thus, I need to run:

```
wandb.tensorboard.patch(root_logdir=log_directory)
wand.init()
writer = SummaryWriter(log_dir=log_directory)

...
My experiment
...

wandb.finish()
writer.close()
wandb.tensorboard.unpatch()
```

However, `unpatch()` is not documented. If it is not called, an error occurs. I only found it because your code is open-source :)

This is the function: https://github.com/wandb/wandb/blob/0bf2ea43770e7349a57fc776aeb16d3035ce4dbf/wandb/integration/tensorboard/monkeypatch.py#L18

This is the documentation: https://docs.wandb.ai/guides/integrations/tensorboard/. Here, only `patch()` is explained. Hope that helps! :)

**Comment by daniel-bogdoll** [comment]
Hey, this works, but the issue is if you put it into a loop:

```
for i in range(10):
        log_directory = "./tb_logs"
        wandb.tensorboard.patch(root_logdir=log_directory)
        run = wandb.init(entity="demonstrations", project="dis")
        writer = SummaryWriter(log_dir=log_directory)
        for step in range(10):
            writer.add_scalar("metric", step, step)
        
        run.finish()
        writer.close()
```

This should also fail on your side :)

**Comment by fdsig** [comment]
Hey, @daniel-bogdoll -- thanks for providing clarifying details here so quickly. To confirm I can reproduce the error message that you observe when this is inside a for loop. I will request a docs update with the `unpatch()` method. So that we can ensure alignment of understanding would you mind sharing what the use case is here -- I.e are you initializing multiple training runs in series (10 in your example above) with new corresponding `writer` and `run` objects?

**Comment by daniel-bogdoll** [comment]
Yeah, for example, just any type of more than one run in a script

**Comment by fdsig** [comment]
Hey @daniel-bogdoll -- thanks for providing information on your use case. I have provided this feedback to our docs team to include `unpatch()` method.

We'll keep you posted on updates and please do reach out on this or anything else wandb realted either by raising an issue or contacting us at support@wandb.com.
